### PR TITLE
[script.module.pysubs2] 1.1.0

### PR DESCRIPTION
**Pull request by bkiziuk**
### Description
Provides pysubs2 library functionality for Kodi addons

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

File: script.module.pysubs2/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pysubs2" name="pysubs2 module" version="1.0.0" provider-name="bkiziuk">
```
Comment (enen92) [inline script.module.pysubs2/addon.xml:2]: Author is the original author unless this is patched

File: script.module.pysubs2/addon.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.module" library="lib" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Provides pysubs2 library functionality for Kodi addons</summary>
+        <description lang="en_GB">Provides pysubs2 library functionality for Kodi addons</description>
+        <disclaimer lang="en_GB" >May contain a bug</disclaimer>
```
Comment (enen92) [inline script.module.pysubs2/addon.xml:10]: Drop news

**Review by enen92 — COMMENTED** [review]
Just some minors. Make your add-on depend on this one (and remove the bundled code) and both are ready to go. I noticed this is version 1.0.0 but pypi already has version 1.1.0. Any reason for not using the latest one available?

File: script.module.pysubs2/addon.xml (hunk 1 above)
Comment (bkiziuk) [reply script.module.pysubs2/addon.xml:2]: Got it.
Regarding the new version, I missed the latest release by a dozen of hours ;-). Let me correct that.

File: script.module.pysubs2/addon.xml (hunk 2 above)
Comment (bkiziuk) [reply script.module.pysubs2/addon.xml:10]: Ok

**Review by enen92 — APPROVED** [review]

**Pull request merged**